docs-search/client-halogen/spago.yaml

```diff
@@ -33,3 +33,12 @@ package:
     - web-html
     - web-storage
     - web-uievents
+  bundle:
+    type: "app"
+    minify: true
+    sourceMaps: true
+    module: Docs.Search.App
+    outfile: "../../output/Docs.Search.IndexBuilder/docs-search-app.js"
```

Comment:
Bizarrely enough, this is where IndexBuilder expects to find the bundled code.
Is there a better way to do this?

https://github.com/purescript/spago/blob/29cd3cad10e0b2dca093ade0a7a2d114186002ba/docs-search/index/src/Docs/Search/IndexBuilder.purs#L411-L412

Reply:
Nevermind, I'm bundling to the bin/ folder now.